File: resume-screening-app/simple_test.py
import requests

# Test health endpoint
try:
    response = requests.get("http://localhost:5000/api/health")
    print(f"Health endpoint status: {response.status_code}")
    print(f"Response: {response.text}")
except Exception as e:
    print(f"Error testing health endpoint: {e}")

# Test ML status endpoint
try:
    response = requests.get("http://localhost:5000/api/ml/status")
    print(f"ML status endpoint status: {response.status_code}")
    print(f"Response: {response.text}")
except Exception as e:
    print(f"Error testing ML status endpoint: {e}")

File: resume-screening-app/frontend/simple_test.mjs
import axios from 'axios';

async function testAPI() {
    try {
        // Test health endpoint
        const healthResponse = await axios.get('http://localhost:5000/api/health');
        console.log("Health Check:", healthResponse.data);

        // Test ML status endpoint
        const mlStatusResponse = await axios.get('http://localhost:5000/api/ml/status');
        console.log("ML Status:", mlStatusResponse.data);

        // Test job skills endpoint (no file upload needed)
        const jobSkillsResponse = await axios.post('http://localhost:5000/job_skills', {
            job_description: 'Looking for a Python developer with experience in Flask and Django.'
        });
        console.log("Job Skills:", jobSkillsResponse.data);

        console.log("All API endpoints are working correctly!");
        
    } catch (error) {
        console.error("Error during API test:", error.response ? error.response.data : error.message);
    }
}

testAPI();
